docs/LANG.md

```diff
@@ -3,51 +3,53 @@
 
 Sigma frontend implements the following pipeline:
 
-SourceCode --> `Parser` --> `Binder` -> `Typer` -> `Specializer` -> ErgoTree
+SourceCode --> `Parser` --> `Binder` -> `Typer` -> `CompiletimeCosting` -> `TreeBuilding` -> ErgoTree
 
 Here:
 - SourceCode  is a string of unicode characters
 - ErgoTree is an intermediate representation which can be processed by Sigma [Interpreter](https://github.com/ScorexFoundation/sigmastate-interpreter/blob/master/src/main/scala/sigmastate/interpreter/Interpreter.scala)
  
 ## Parser
-`SigmaParser` takes a string and produces abstract syntax tree, AST, of a Sigma expression.
+`SigmaParser` takes a string and produces abstract syntax tree, AST, of a Sigma expression represent by `Value` type in Scala.
 
 In case of any errors it throws `ParserException`
 
 ## Binder
 `SigmaBinder` takes an AST of successfully parsed Sigma expression and resolves 
 global variables and predefined functions be looking up in the provided environment.
```

Comment:
`SigmaBinder` takes an AST of successfully parsed Sigma expression and resolves global variables and predefined functions that are looked up in the provided environment.

Reply:
fixed in next pr